hack/code/bandwidth_gen.go

```diff
@@ -71,6 +72,31 @@ func main() {
 			bandwidthData := row.FirstChild.NextSibling.NextSibling.NextSibling.FirstChild.Data
 			bandwidth[instanceTypeData] = int64(lo.Must(strconv.ParseFloat(bandwidthData, 64)) * 1000)
 		}
+		for uri, selector := range uriSelectors {
+			func() {
+				response := lo.Must(http.Get(uri))
+				defer response.Body.Close()
+
+				doc := lo.Must(goquery.NewDocumentFromReader(response.Body))
+				// This query looks
```

Comment:
We should add some comments throughout this function to describe what this function is doing and how it is querying the webpage